docs/core/install/localized-intellisense.md

```diff
@@ -0,0 +1,58 @@
+---
+title: Install localized IntelliSense files
+description: Learn how to set up your development machine to use localized IntelliSense files.
+author: mairaw
+ms.author: mairaw
+ms.date: 12/12/2019
+---
+# How to install localized IntelliSense files for .NET Core projects
+
+IntelliSense is a code-completion aid that is available in different integrated development environments (IDEs), such as Visual Studio. By default, when you're developing .NET Core projects, the SDK only includes the English version of the IntelliSense files. This article explains how to install the localized version of those files and modify the Visual Studio installation to use a different language.
+
+## Prerequisites
+
+- [.NET Core 3.0 SDK](https://dotnet.microsoft.com/download/dotnet-core) or later versions
+- [Visual Studio 2019 16.3 or later versions](https://visualstudio.microsoft.com/downloads/?utm_medium=microsoft&utm_source=docs.microsoft.com&utm_campaign=inline+link&utm_content=download+vs2019)
+
+## Download and install the localized IntelliSense files
+
+1. Go to the [Download IntelliSense files](https://dotnet.microsoft.com/download/dotnet-core) page.
```

Comment:
I am assuming that this link we will eventualy update to the new Intellisense page that is coming up.

Reply:
Yes 😄

Reply:
I've updated to what I think it will be the final URL. Looks right?